app/controllers/comments_controller.rb

```diff
@@ -6,9 +6,9 @@ def create
   end
 
   def update
-    @post = Post.find(params[:post_id])
-    @comment = @post.comments.find(params[:id])
-    @comment.likes = @comment.likes + 1
+    @post = Post.find(params[:id])
+    @comment = @post.comments.find(params[:post_id]) # works but ids are flipped, needs to be fixed
```

Comment:
ID's flipped as in it picks up the wrong one ?

Reply:
Our post_id and id were being passed the wrong way around. But we fixed this in the latest version.